### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What and why?

Release `v1.3.0`: 

* #510
* #512 
* #515 
* #516 
* #513

### CODE REVIEW

File: src/commands/lambda/__tests__/instrument.test.ts [hunk 1]
```diff
@@ -1318,7 +1318,7 @@ UpdateFunctionConfiguration -> arn:aws:lambda:sa-east-1:123456789012:function:la
 }
 TagResource -> arn:aws:lambda:sa-east-1:123456789012:function:lambda-hello-world
 {
-  \\"dd_sls_ci\\": \\"v1.2.0\\"
+  \\"dd_sls_ci\\": \\"v1.3.0\\"
```
Comment (lefebvree) [inline src/commands/lambda/__tests__/instrument.test.ts:1321]: A bit tedious to update every time, we should look for a way to mock the version there in the future

**Review by lefebvree — APPROVED** [review]

File: src/commands/lambda/__tests__/instrument.test.ts (hunk 1 above)
Comment (juan-fernandez) [reply src/commands/lambda/__tests__/instrument.test.ts:1321]: agreed. Would it be as simple as reading `package.json` from the root?

Comment (lefebvree) [reply src/commands/lambda/__tests__/instrument.test.ts:1321]: Yes, it seems it was already done in some tests. Created a small PR to update it: https://github.com/DataDog/datadog-ci/pull/520

Comment (juan-fernandez) [reply src/commands/lambda/__tests__/instrument.test.ts:1321]: nice, thanks

**Pull request merged**